tools/tcpretrans.py

```diff
@@ -70,6 +74,24 @@
 };
 BPF_PERF_OUTPUT(ipv6_events);
 
+// separate flow keys per address family
+struct ipv4_fl_k_t {
+    // XXX: switch some to u32's when supported
```

Comment:
Could you just use ipv4_flow_key_t and below ipv6_flow_key_t? I thought the length of the struct name still not too long?